tests/test_document.py

```diff
@@ -158,14 +173,22 @@ def test_has_permission(pdf_document):
 
 def test_info_date(pdf_document):
     date = pdf_document.info_date("CreationDate")
-    assert date.astimezone(timezone.utc) == datetime(
-        2020, 3, 26, 1, 19, 50, tzinfo=timezone.utc
-    )
+    if version() < (0, 46, 0):
+        assert date.astimezone(timezone.utc) == datetime(
+            2020, 3, 25, 21, 19, 50, tzinfo=timezone.utc
+        )
```

Comment:
Are-you sure the wrong date is related with the poppler version? On my side, the date is the same with older poppler version.... Are you sure you computer has the right timezone?

Reply:
I'll take a closer look, I'm using inside an AWS SageMaker notebook, so it might not be set. However, maybe there is  somewhere else the timezone is being used incorrectly.